### ISSUE TITLE
husky 에러

### ISSUE THREAD

**Issue by 5o-hyun** [swyp-mema/mema-client#38]
mac을 사서 (새로운 환경에서 ) 커밋을 보냈는데 husky가 not found가 뜨는 에러를 발견했습니다!
구글링해보니 다음과같은 명령어를 치라해서 husky 설정을 초기화하고 설치하니 되었습니다!
>npx husky-init && npm install
<img width="343" alt="image" src="https://github.com/user-attachments/assets/ce1ef096-47f9-4a56-8fda-d1e9a5e69d31">

해결된 부분을 보니 package.json의 prepare부분이 바뀐것을 확인할 수 있었습니다
아마 해당부분을 바꿔줘야 모든 환경에서 동작할수있는것같습니다

맥북에서 작업을 하지는 않을거라 상관없긴한데 이슈발견되어 해당부분 알려드리려고 작성했습니다! 
지금은 작업때문에 바쁘니 15일에 1차배포 후 나중에는 변경해야 할것같은데 코멘트남겨주세요!

**Comment by YerangPark** [comment]
혹시 `npm install` 이후 커밋하셨을 때 에러가 발생하셨나요?

**Comment by 5o-hyun** [comment]
> 혹시 `npm install` 이후 커밋하셨을 때 에러가 발생하셨나요?

네 npm i 했는데 설치할게없다고뜨고(설치완료되었다고뜹니다) commit하려고보면 husky not found가 뜹니다.
그래서 구글링해보니 초기화하고 깔아보라길래 husky 초기화하고 npm i 하니 에러발생없이 잘 커밋되었습니다!

**Comment by YerangPark** [comment]
왜인지 제 환경에서 새로 클론받아서 테스트해보았을 때는 문제가 없었는데 문제를 찾아봐야겠네요

제 테스트 시퀀스는 이랬습니다.
1. 프로젝트 클론
2. npm install
3. 아무 파일이나 임의로 수정 후 커밋 메세지에 "test"만 적고 커밋
![image](https://github.com/user-attachments/assets/8c3fe8c6-3782-42cf-a922-c42f91b0930f)
-> 에러 발생

**Comment by 5o-hyun** [comment]
> 왜인지 제 환경에서 새로 클론받아서 테스트해보았을 때는 문제가 없었는데 문제를 찾아봐야겠네요
> 
> 제 테스트 시퀀스는 이랬습니다.
> 
> 1. 프로젝트 클론
> 2. npm install
> 3. 아무 파일이나 임의로 수정 후 커밋 메세지에 "test"만 적고 커밋
>    ![image](https://private-user-images.githubusercontent.com/27879946/394133070-8c3fe8c6-3782-42cf-a922-c42f91b0930f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM4MTcxNDAsIm5iZiI6MTczMzgxNjg0MCwicGF0aCI6Ii8yNzg3OTk0Ni8zOTQxMzMwNzAtOGMzZmU4YzYtMzc4Mi00MmNmLWE5MjItYzQyZjkxYjA5MzBmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjEwVDA3NDcyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTliZjBlYzFjY2MxYTI1MjY0YWE0MjIyMGY0NmIxMGVhM2Q3ZDczNmU0MzI4MWU2ZTYzNzJkNjU1MjFkNGIyZGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.VMu47j8KviG2_47QP3LQcza99-aNRXtBpD8AVznCk2M)
>    -> 에러 발생

엇 지금 에러 발생하시는건가요?? 현재 문제발생 안된다고 하지않으셨나용? ㅇㅂㅇ

**Comment by YerangPark** [comment]
> > 왜인지 제 환경에서 새로 클론받아서 테스트해보았을 때는 문제가 없었는데 문제를 찾아봐야겠네요
> > 제 테스트 시퀀스는 이랬습니다.
> > 
> > 1. 프로젝트 클론
> > 2. npm install
> > 3. 아무 파일이나 임의로 수정 후 커밋 메세지에 "test"만 적고 커밋
> >    ![image](https://private-user-images.githubusercontent.com/27879946/394133070-8c3fe8c6-3782-42cf-a922-c42f91b0930f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM4MTcxNDAsIm5iZiI6MTczMzgxNjg0MCwicGF0aCI6Ii8yNzg3OTk0Ni8zOTQxMzMwNzAtOGMzZmU4YzYtMzc4Mi00MmNmLWE5MjItYzQyZjkxYjA5MzBmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjEwVDA3NDcyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTliZjBlYzFjY2MxYTI1MjY0YWE0MjIyMGY0NmIxMGVhM2Q3ZDczNmU0MzI4MWU2ZTYzNzJkNjU1MjFkNGIyZGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.VMu47j8KviG2_47QP3LQcza99-aNRXtBpD8AVznCk2M)
> >    -> 에러 발생
> 
> 엇 지금 에러 발생하시는건가요?? 현재 문제발생 안된다고 하지않으셨나용? ㅇㅂㅇ

"test"만 적으면 커밋 메세지 format에 안맞는거라서 에러 발생 되는게 정상 시퀀스입니다

**Comment by 5o-hyun** [comment]
> > > 왜인지 제 환경에서 새로 클론받아서 테스트해보았을 때는 문제가 없었는데 문제를 찾아봐야겠네요
> > > 제 테스트 시퀀스는 이랬습니다.
> > > 
> > > 1. 프로젝트 클론
> > > 2. npm install
> > > 3. 아무 파일이나 임의로 수정 후 커밋 메세지에 "test"만 적고 커밋
> > >    ![image](https://private-user-images.githubusercontent.com/27879946/394133070-8c3fe8c6-3782-42cf-a922-c42f91b0930f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM4MTcxNDAsIm5iZiI6MTczMzgxNjg0MCwicGF0aCI6Ii8yNzg3OTk0Ni8zOTQxMzMwNzAtOGMzZmU4YzYtMzc4Mi00MmNmLWE5MjItYzQyZjkxYjA5MzBmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjEwVDA3NDcyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTliZjBlYzFjY2MxYTI1MjY0YWE0MjIyMGY0NmIxMGVhM2Q3ZDczNmU0MzI4MWU2ZTYzNzJkNjU1MjFkNGIyZGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.VMu47j8KviG2_47QP3LQcza99-aNRXtBpD8AVznCk2M)
> > >    -> 에러 발생
> > 
> > 
> > 엇 지금 에러 발생하시는건가요?? 현재 문제발생 안된다고 하지않으셨나용? ㅇㅂㅇ
> 
> "test"만 적으면 커밋 메세지 format에 안맞는거라서 에러 발생 되는게 정상 시퀀스입니다

아하 저는 test: gittest -#3 이런식으로 원래 맞추기로 했던 형태로 했는데 안됬습니다. 근데 에러메세지가아니라 그냥 not found가 떳어요. 형태문제가아니라 아예 husky를 못찾는에러같아요

**Comment by YerangPark** [comment]
이상하네요 같은 시퀀스로 진행했을 때 현상이 정반대인데.. 환경차이가 있으려나요...??😵‍💫 여유가 좀 생기면 그 때 디버깅 해보시죠

**Comment by YerangPark** [comment]
아 수정해서 커밋을 하셨군요! 재설치 안하고 package.json만 수정해서 해결이 안되나요?? 제가 9버전 기준으로 보고 환경 설정 해놓았던 건데 혹시나 문제가 생길까봐요

**Comment by 5o-hyun** [comment]
네 그냥 npm i 하면 계속 not found만 뜨고 (다 받았는데도 Not found가 뜸), 허스키 초기화 후 npm i 하니 잘되는 상황입니다.

**Comment by 5o-hyun** [comment]
이부분 혹시 방화벽때문에 생길수도있는 문제일까요?? 제가 최근에 해커톤갔다왔는데 거기서 localhost:8080으로 백에 요청하니 안되는 이슈가 있었거든요. 근데 다른컴터는되는데 제 컴에서만 안됬었어요. 핑도 돌아가는거 확인했는데,, 그래서 방화벽 문제같은데 혹시 그거때문일수도있나해서요

**Comment by YerangPark** [comment]
테스트했을 당시에 제 방화벽이 열려있긴 했습니다. 방화벽 문제가 맞는지 확인하려면 꺼보고 켜보고 테스트를 해봐야 알 것 같습니다.
